channel/consensus_channel/consensus_channel.go

```diff
@@ -894,7 +894,10 @@ func (c *ConsensusChannel) Clone() *ConsensusChannel {
 	for i, p := range c.proposalQueue {
 		clonedProposalQueue[i] = p.Clone()
 	}
-	d := ConsensusChannel{c.MyIndex, c.fp.Clone(), c.Id, c.OnChainFunding.Clone(), c.current.clone(), clonedProposalQueue}
+	d := ConsensusChannel{
+		MyIndex: c.MyIndex, fp: c.fp.Clone(),
+		Id: c.Id, OnChainFunding: c.OnChainFunding.Clone(), current: c.current.clone(), proposalQueue: clonedProposalQueue,
+	}
```

Comment:
Nit: not opposed to this change, but seems very unrelated to the PR. Also, if you are forming literals with named arguments, the order of the fields doesn't matter... so I'm not really understanding the benefit?

Reply:
The order of the fields of `consensus_channel` was changed to match that of channel. As a result, this struct construction started failing since the values were not named. Happy to move https://github.com/statechannels/go-nitro/pull/1239/commits/1ff9feb595192863d760c7d4b72a9adb6ee8a0fe to a separate PR if there is a strong preference.